environment/src/main/java/com/sequenceiq/environment/events/EnvironmentStructuredFlowEventFactory.java

```diff
@@ -40,11 +41,11 @@ public CDPStructuredFlowEvent createStructuredFlowEvent(Long resourceId, FlowDet
     @Override
     public CDPStructuredFlowEvent createStructuredFlowEvent(Long resourceId, FlowDetails flowDetails, Boolean detailed, Exception exception) {
         Environment environment = environmentService.findEnvironmentByIdOrThrow(resourceId);
-        String resourceType = CDPStructuredFlowEvent.class.getSimpleName();
+        String resourceType = CloudbreakEventService.ENVIRONMENT_RESOURCE_TYPE;
```

Comment:
It indicates the type of the given operation and JSON deserialisation logic uses that field. I wouldn't change it, because it has already fixed previously to work properly. Please test it with querying events from the backend, because I am sure this change would break get operation on structured events.